pkgs/dev_mcp_host/devtools_options.yaml

```diff
@@ -0,0 +1,4 @@
+description: This file stores settings for Dart & Flutter DevTools.
+documentation: https://docs.flutter.dev/tools/devtools/extensions#configure-extension-enablement-states
+extensions:
+  - dev_mcp_host: true
```

Comment:
A fair number of these files are missing trailing newlines, would be nice to get those in